metrotranscom/doorway
# Fix/add filter translations

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description
Added translations for most of the filter text.

**Still need to add in the labels/values: studio, any, and the DAHLIA link text. This will be done in another PR. (I added the translations in this PR, but need to use t() to reference them in a new PR.)**

Listings filter:
![image](https://github.com/metrotranscom/doorway/assets/12400715/6763416e-ba04-48ac-b3ad-c854da2700c7)

Landing filter:
![image](https://github.com/metrotranscom/doorway/assets/12400715/bf4d0661-e4dd-4a80-9370-dbd3b4e27206)

## How Can This Be Tested/Reviewed?
View the files for the changes. Can also run the public site and change the languages on the Landing & Listings page & verify that the filters are translated.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
